### DISCUSSION TITLE
No track details...

### DISCUSSION THREAD

**Discussion by WolfgangZ42** [devemux86/cruiser#118] [Q&A] [answered]
By saving my GPS data after a tour I saw "Trackdetails" as offering:

<img width="400" src="https://user-images.githubusercontent.com/121806238/230426279-054de716-9ed7-4efc-b0e1-73d4d92a716c.png"/>  

but the output is extremly short:

<img width="400" src="https://user-images.githubusercontent.com/121806238/230426304-b4da40ff-3d4d-4e1d-bab6-3b34587cddc4.png"/>  

I don't miss anything - was just curious ...

**Comment by devemux86** [comment, marked as answer]
This option is not supposed to appear yet, it is still a work in progress!

In the future, Cruiser will display details about the recorded GPS track. :slightly_smiling_face:

**Reply by 0709wiwiwi** [reply]
Not limited to track records only but also to display the gpx(nav)track details like some other apps do ?
![Gpx navtrack details](https://user-images.githubusercontent.com/54167201/230786348-375c9bf6-75fc-49bd-8e8e-1986c11d44ee.png)

[Cruiser_Brouter_shaping_Kodela.zip](https://github.com/devemux86/cruiser/files/11185933/Cruiser_Brouter_shaping_Kodela.zip)

**Reply by 0709wiwiwi** [reply]
**Track Details.** 
**Cruiser - Settings - Routing - Options - Route duration = SET Route.**
At import of a timestamped gpx navtrack, the track duration in both "popup up" and in the routing instructions timing list is not shown correctly by the available trackpoint timestamps.  The expected shown total track time is 5:25:45 

- Import in Locus & in gpx viewer do correctly display the total track time..
Above find the example gpx file to test.

**Reply by devemux86** [reply]
@0709wiwiwi thanks for the report.

_This discussion is (answered) on a different subject, please create new topics to discuss different things._ :slightly_smiling_face: 

---

About the report, since **gpx navigation tracks** may or may not have time data (or it may not be complete),
we use the selected profile and its average speed to calculate the time for imported **gpx navigation tracks**.

**Reply by 0709wiwiwi** [reply]
Done.
https://github.com/devemux86/cruiser/discussions/124#discussion-5062874
